CLID-66,CLID-80: changes README.md (#837)
openshift/oc-mirror: v2/README.md

- Phase 1 (--generate): using a delete set configuration as an input, oc-mirror discovers all images that needed to be deleted. These images are included in a delete-images file to be consumed as input in the second phase.
```
oc-mirror delete -c ./delete-isc.yaml --generate --workspace file:///home/<user>/oc-mirror/delete1 --delete-id delete1-test docker://localhost:6000 --v2
```

- Phase 2: using the file generated in first phase, oc-mirror will delete all manifests specified on this file on the destination specified in the command line. It is up to the container registry to run the garbage collector to clean up all the blobs which are not referenced by a manifest. Deleting only manifests is safer since blobs shared between more than one image are not going to be deleted.
```
oc-mirror delete --delete-yaml-file /home/<user>/oc-mirror/delete1/working-dir/delete/delete-images-delete1-test.yaml docker://localhost:6000 --v2
```

## Development
This section will give an overview about oc-mirror v2 architecture. Pull requests are always welcome in the oc-mirror repository.

### Architecture
![Alt text](./assets/architecture.png)

### Image Set Configuration Specs

> [Here](https://github.com/openshift/oc-mirror/blob/e7889a7ec70dd66b0d6a7ba6dedc3e4b93ebf4de/v2/pkg/api/v2alpha1/type_config.go#L16) it is possible to find the specification of the Image Set Configuration with all available fields.

Reviewer: ```suggestion
[Link](https://github.com/openshift/oc-mirror/blob/e7889a7ec70dd66b0d6a7ba6dedc3e4b93ebf4de/v2/pkg/api/v2alpha1/type_config.go#L16) that shows details about the specification of the Image Set Configuration with all available fields.
```

Author: Done ✅ 

Thanks for the suggestion.

## Testing
Currently oc-mirror has unit tests and integration tests. 

To run unit test:

```
make test
```

To run integration tests:
```
make test-integration
```

> ## To Know More About

Reviewer: ```suggestion
## Find out more
```

Author: Done ✅ 

Thanks for the suggestion.
